exercises/isogram/isogram-example.factor

```diff
@@ -0,0 +1,5 @@
+USING: kernel ascii sets sequences ;
+IN: isogram
+
+: isogram? ( str -- ? )
+  >lower [ letter? ] filter dup members [ length ] bi@ = ;
```

Comment:
This pull request on the whole looks excellent, but I do have a cleaner and more Factor-y `isogram?` definition:

```factor
USING: kernel sets ;
: isogram? ( string -- ? )
  dup "" set-like = ;
```